.vscode/tasks.json

```diff
@@ -5,9 +5,9 @@
   "tasks": [
     {
       "label": "typecheck",
-      "type": "shell",
+      "type": "process",
```

Comment:
I don't remember exactly, but I think this was causing problems on Windows, maybe? Is there a reason to change from `shell` to `process`?

Reply:
That is why I've changed it to `process`. This is what you're thinking about: https://github.com/rzk-lang/vscode-rzk/commit/c30842c5175f6df027e9e1298a00fe99e1cf49c2